core/build.gradle.kts

```diff
@@ -295,6 +294,40 @@ tasks.withType<org.jetbrains.dokka.gradle.AbstractDokkaLeafTask> {
 
 // endregion
 
+// region experimental artifact
+val experimentalJar by tasks.registering(Jar::class) {
+    archiveBaseName.set("experimental")
+    from(experimental.output)
+}
+
+val experimentalJarSources by tasks.registering(Jar::class) {
+    dependsOn("kspKotlin")
```

Comment:
If it's dependent on the generated sources, it should probably also depend on the task `processKDocsMain`, right?